scans/response.py

```diff
@@ -10,19 +26,11 @@ def without_security_headers(self):
 
     def with_security_headers(self):
         security_response_dict = self.response_dict.copy()
-        security_response_dict["headers"] = {
-            "Access-Control-Allow-Methods": "GET, POST, OPTIONS, HEAD",
-            "Access-Control-Allow-Origin": "*",
-            "Access-Control-Max-Age": "86400",
-            "Cache-control": "no-store",
-            "Pragma": "no-cache",
-            "Content-Security-Policy": "default-src 'none'; frame-ancestors 'none'; script-src 'none'; upgrade-insecure-requests",
-            "Referrer-Policy": "no-referrer",
-            "X-Content-Type-Options": "nosniff",
-            "X-Download-Options": "noopen",
-            "X-Frame-Options": "DENY",
-            "X-Permitted-Cross-Domain-Policies": "none",
-            "X-XSS-Protection": "1; mode = block",
-            "Strict-Transport-Security": "max-age = 15768000; includeSubDomains"
-        }
+
+        # initialize headers if not already
+        if not security_response_dict.get("headers"):
+            security_response_dict["headers"] = {}
+
+        for header_name, header_value in self.SECURITY_HEADERS.items():
```

Comment:
You can use the dict constructor to perform this operation:
```python
>>> a = { 'prime': 5 }
>>> b = dict(a)
>>> b['prime'] = 7
>>> a
{'prime': 5}
>>> b
{'prime': 7}
```

Reply:
I'm merging this PR, but still interested in optimization opportunity here